src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/DeparturesWidget.tsx

```diff
@@ -49,7 +49,7 @@ export const DeparturesWidget = ({
   const {location} = useGeolocationState();
   const {state, loadInitialDepartures, searchDate} = useFavoriteDepartureData();
 
-  useEffect(() => loadInitialDepartures(), [favoriteDepartures]);
+  useEffect(() => loadInitialDepartures(), [loadInitialDepartures]);
```

Comment:
Could this break functionality where the data for the widget is updated when `favoriteDepartures` changes?

Reply:
That is handled, as this is the signature for `loadInitialDepartures`:
```tsx
const loadInitialDepartures = useCallback(
  () =>
    dispatch({
      type: 'LOAD_INITIAL_DEPARTURES',
      locationOrStopPlace,
      favoriteDepartures,
    }),
  [locationOrStopPlace?.id, favoriteDepartures],
);
```

Reply:
Great!